File: library/src/main/java/com/alamkanak/weekview/EmptyViewLongPressListener.java
package com.alamkanak.weekview;

import java.util.Calendar;

public interface EmptyViewLongPressListener {
    /**
     * Similar to {@link com.alamkanak.weekview.WeekView.EmptyViewClickListener} but with long press.
     *
     * @param time: {@link Calendar} object set with the date and time of the long pressed position on the view.
     */
    void onEmptyViewLongPress(Calendar time);
}

File: library/src/main/java/com/alamkanak/weekview/EmptyViewLongPressListener.kt
package com.alamkanak.weekview

import java.util.Calendar

interface EmptyViewLongPressListener {
    /**
     * Similar to [com.alamkanak.weekview.EmptyViewClickListener] but with long press.
     *
     * @param time: [Calendar] object set with the date and time of the long pressed position on the view.
     */
    fun onEmptyViewLongPress(time: Calendar)
}